woodwork/table_accessor.py

```diff
@@ -145,6 +146,27 @@ def __getattr__(self, attr):
         else:
             raise AttributeError(f"Woodwork has no attribute '{attr}'")
 
+    def __getitem__(self, key):
+        if isinstance(key, list):
+            index = pd.Index(key)
+            columns = self._dataframe.columns
+            diff = index.difference(columns)
+            in_columns = diff.isin(columns).all()
```

Comment:
Anything in `diff` shouldn't be in columns, so the only way in_columns is True is if `diff` is empty, so at line 156, it can just be
```python
if diff:
    raise KeyError(...)
```